### PR TITLE
2004/web 4995 kibana document timestamps do not show milliseconds final approach

### PR DESCRIPTION
**Pull request by collin-mcgrath**
https://github.com/Invoca/web/pull/12612
https://github.com/Invoca/aggregate/pull/51

### CODE REVIEW

File: spec/query/match_condition_spec.rb [hunk 1]
```diff
@@ -33,14 +33,14 @@
       end
 
       it "converts time to iso8601 and returns range" do
-        time = Time.utc(2018, 12, 27, 20, 10).in_time_zone("Pacific Time (US & Canada)")
+        time = Time.utc(2018, 12, 27, 20, 10, 0o0, 123).in_time_zone("Pacific Time (US & Canada)")
```
Comment (dtognazzini) [inline spec/query/match_condition_spec.rb]: Why the use of octal here? (same question elsewhere)

**Review by dtognazzini — APPROVED** [review]

File: spec/query/match_condition_spec.rb (hunk 1 above)
Comment (collin-mcgrath) [reply spec/query/match_condition_spec.rb]: Rubocop formatted it like this after Hound barked (in another PR). I don't see any specific Invoca style preferences for this one so I just went with it.

Comment (collin-mcgrath) [reply spec/query/match_condition_spec.rb]: Or (lol) are you saying this should just be `0`? If so, d'oh and I agree.

Comment (dtognazzini) [reply spec/query/match_condition_spec.rb]: Ah, ok. Did  you use `00`? That's an octal number in Ruby. It's best to keep with decimal a la `0`.

Comment (ColinDKelley) [reply spec/query/match_condition_spec.rb]: Yeah, you must have had leading 0 in there, which is pretty common for dates. But unfortunately Ruby carried across the format from C that says that is an octal constant.
So: get rid of the leading 0 and also the 0c!

Comment (collin-mcgrath) [reply spec/query/match_condition_spec.rb]: Resolved.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 # Changelog
 All notable changes to this project will be documented in this file
 
+## [2.0.0] - 2020-04-16
```
Comment (omeedrj) [inline CHANGELOG.md]: nit: Out of date lol

**Review by omeedrj — APPROVED** [review]
👍

File: CHANGELOG.md (hunk 2 above)
Comment (collin-mcgrath) [reply CHANGELOG.md]: Bah. I knew that would be out of date _immediately_. Fixed.

**Comment by collin-mcgrath** [comment]
Looks like I'm missing the ticket number in these commits since I apparently don't have the git hooks installed in this repo. I can rebase and fix that one this gets approved.

**Comment by collin-mcgrath** [comment]
Rebasing to consolidate commits and commit messages, as well as address conflicts.

**Pull request merged**